scripts/e2e/hack/Dockerfile

```diff
@@ -0,0 +1,100 @@
+# Build the Go binaries using a multi-stage build phase named "golang"
+FROM debian:jessie
+LABEL maintainer="Hui  Luo <luoh@vmware.com>"
+
+WORKDIR /workspace
+RUN mkdir -p /workspace
+ENV WORKSPACE=/workspace \
+    TERM=xterm
+
+# common util tools
+# https://github.com/GoogleCloudPlatform/gsutil/issues/446 for python-openssl
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    file \
+    git \
+    jq \
+    mercurial \
+    openssh-client \
+    pkg-config \
+    python \
+    python-dev \
+    python-openssl \
+    python-pip \
+    rsync \
+    unzip \
+    wget \
+    xz-utils \
+    zip \
+    zlib1g-dev \
+    && apt-get clean \
+    && python -m pip install --upgrade pip setuptools wheel
+
+# Go standard envs
+ENV GOPATH /go
+ENV PATH /usr/local/go/bin:$PATH
+ENV PATH $GOPATH/bin:$PATH
+
+# install go
+ENV GO_VERSION=1.10.3
+ENV GO_TARBALL "go${GO_VERSION}.linux-amd64.tar.gz"
+RUN wget -q "https://storage.googleapis.com/golang/${GO_TARBALL}" && \
+    tar xzf "${GO_TARBALL}" -C /usr/local && \
+    rm "${GO_TARBALL}"
+
+#
```

Comment:
You could have also built this image from either the official DinD image or Golang image instead of Debian and installing all of these yourself.